### PR TITLE
Various bug fixes

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
1. upgrade to latest dscp versions so that we can make demo work offline
2. remove auth so demo works offline
3. fix docker scripts so separate projects are handled correctly
4. fix file handling using data URLs so cached data renders correctly
5. introduce a reset if the lastTokenId decreases so that resets are easier
6. layout bug on am negotiation
7. increase poll frequency and reduce manufacture time to make demo more snappy
8. add in charlie node to prevent split brain situations
9. remove pointless helm chart. No idea why we have this for a local demo

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -29,77 +29,6 @@ jobs:
           echo "::set-output name=REPO_NAME::$REPO_NAME"
           echo "::set-output name=ORG_NAME::$ORG_NAME"
```
Comment (dblane-digicatapult) [inline .github/workflows/release.yml:30]: Change set output syntax

File: .github/workflows/run_tests.yml [hunk 2]
```diff
@@ -20,77 +20,6 @@ jobs:
           ORG_NAME=$(echo "${{ github.event.repository.owner.name }}" | tr '[:upper:]' '[:lower:]')
           echo "::set-output name=REPO_NAME::$REPO_NAME"
           echo "::set-output name=ORG_NAME::$ORG_NAME"
```
Comment (dblane-digicatapult) [inline .github/workflows/run_tests.yml:22]: Change set output syntax

File: docker/docker-compose-am.yml [hunk 3]
```diff
@@ -2,24 +2,28 @@ version: '3'
...
       driver: default
   am:
+    name: am
     ipam:
       driver: default
 
 services:
   node-bob:
-    image: ghcr.io/digicatapult/dscp-node:v3.0.1
+    image: ghcr.io/digicatapult/dscp-node:v4.5.1
```
Comment (dblane-digicatapult) [inline docker/docker-compose-am.yml:19]: Double check pinned version.

File: docker/docker-compose-am.yml [hunk 4]
```diff
@@ -34,7 +38,7 @@ services:
     networks: ['chain', 'am']
 
   ipfs-bob:
-    image: ghcr.io/digicatapult/dscp-ipfs:v2.0.0
+    image: ghcr.io/digicatapult/dscp-ipfs:v2.9.1
```
Comment (dblane-digicatapult) [inline docker/docker-compose-am.yml:41]: Double check pinned version.

File: docker/docker-compose-am.yml [hunk 5]
```diff
@@ -43,11 +47,12 @@ services:
       - ../data/am/ipfs:/ipfs
     environment:
       - NODE_HOST=node-bob
+      - IPFS_API_HOST=localhost
     restart: on-failure
     networks: ['ipfs', 'am']
 
   api-bob:
-    image: ghcr.io/digicatapult/dscp-api:v4.0.0
+    image: ghcr.io/digicatapult/dscp-api:v4.8.2
```
Comment (dblane-digicatapult) [inline docker/docker-compose-am.yml:55]: Double check pinned version

File: scripts/check-version.sh [hunk 6]
```diff
@@ -4,22 +4,13 @@
...
-     [ "v$PACKAGE_VERSION" != "$HELM_VALUES_TAG_VERSION" ] ||
-     [ "$PACKAGE_VERSION" != "$HELM_CHART_VERSION" ] ||
-     [ "$PACKAGE_VERSION" != "$HELM_CHART_APP_VERSION" ]; then
+  local PACKAGE_VERSION=$(yq eval -o y '.version' ./package.json)
+  local PACKAGE_LOCK_VERSION=$(yq eval -o y '.version' ./package-lock.json)
+
+  if [ "$PACKAGE_VERSION" != "$PACKAGE_LOCK_VERSION" ]; then
     echo "Inconsistent versions detected"
     echo "PACKAGE_VERSION: $PACKAGE_VERSION"
     echo "PACKAGE_LOCK_VERSION: $PACKAGE_LOCK_VERSION"
```
Comment (dblane-digicatapult) [inline scripts/check-version.sh:13]: What is this supposed to do?  the syntax is completely wrong.

File: scripts/check-version.sh [hunk 7]
```diff
@@ -49,7 +40,7 @@ function check_version_greater () {
 # Get published git tags that match semver regex with a "v" prefix then remove the "v" character
 PUBLISHED_VERSIONS=$(git tag | grep "^v[0-9]\+\.[0-9]\+\.[0-9]\+\(\-[a-zA-Z-]\+\(\.[0-9]\+\)*\)\{0,1\}$" | sed 's/^v\(.*\)$/\1/')
 # Get the current version from node Cargo.toml
-CURRENT_VERSION=$(yq eval '.version' ./package.json)
+CURRENT_VERSION=$(yq eval -o y '.version' ./package.json)
 
 if check_version_greater "$CURRENT_VERSION" "$PUBLISHED_VERSIONS"; then
   echo "##[set-output name=VERSION;]v$CURRENT_VERSION"
```
Comment (dblane-digicatapult) [inline scripts/check-version.sh:46]: Change syntax.

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

File: docker/docker-compose-am.yml (hunk 5 above)
Comment (mattdean-digicatapult) [reply docker/docker-compose-am.yml:55]: worked when I gave it at the event

File: docker/docker-compose-am.yml (hunk 4 above)
Comment (mattdean-digicatapult) [reply docker/docker-compose-am.yml:41]: worked when I gave it at the event

Comment (mattdean-digicatapult) [reply docker/docker-compose-am.yml:41]: worked at event

**Review by dblane-digicatapult — APPROVED** [review]

**Pull request merged**